app/src/test/components/track.test.tsx

```diff
@@ -40,103 +40,164 @@ beforeAll(() => {
+    expect(_.size(state.task.items[0].labels)).toEqual(0)
+    expect(_.size(state.task.tracks)).toEqual(2)
   })
 
-  // Terminate the track by key
-  dispatch(action.goToItem(2))
-  mouseMoveClick(label2d, 1, 30)
-  fireEvent(
-    getByText('Delete'),
-    new MouseEvent('click', {
-      bubbles: true,
-      cancelable: true
-    })
-  )
-  state = getState()
-  expect(_.size(state.task.items[2].labels)).toEqual(1)
-  expect(_.size(state.task.items[1].labels)).toEqual(1)
-  expect(_.size(state.task.tracks[trackIds[0]].labels)).toEqual(2)
-
-  // Delete the track by key
-  dispatch(action.goToItem(6))
-  expect(_.size(state.task.items[6].labels)).toEqual(3)
-  Session.dispatch(selectLabel(
-    state.user.select.labels, 6,
-    state.task.tracks[trackIds[3]].labels[6]))
-  fireEvent(
-    getByText('Delete'),
-    new MouseEvent('click', {
-      bubbles: true,
-      cancelable: true
-    })
-  )
-  state = getState()
-  expect(_.size(state.task.items[6].labels)).toEqual(2)
-  expect(_.size(state.task.tracks)).toEqual(3)
-
-  // Merge tracks
-  dispatch(action.goToItem(1))
-  mouseMoveClick(label2d, 1, 30)
-  state = getState()
-  fireEvent(
-    getByText('Track-Link'),
-    new MouseEvent('click', {
-      bubbles: true,
-      cancelable: true
-    })
-  )
-
-  dispatch(action.goToItem(4))
-  Session.dispatch(selectLabel(
-    state.user.select.labels, 4,
-    state.task.tracks[trackIds[2]].labels[4]))
-  state = getState()
-  fireEvent(
-    getByText('Finish Track-Link'),
-    new MouseEvent('click', {
-      bubbles: true,
-      cancelable: true
-    })
-  )
-  state = getState()
-  expect(_.size(state.task.tracks)).toEqual(2)
+  test('Linking tracks', () => {
+    const label2d = canvasRef.current as Label2dCanvas
+
+    const toolbarRef: React.Ref<ToolBar> = React.createRef()
+    const { getByText } = render(
+      <ToolBar
+        ref={toolbarRef}
+        categories={null}
+        attributes={[]}
+        labelType={'labelType'}
+      />
+    )
+    expect(toolbarRef.current).not.toBeNull()
+    expect(toolbarRef.current).not.toBeUndefined()
+    if (toolbarRef.current) {
+      toolbarRef.current.componentDidMount()
+    }
+
+    const itemIndices = [0, 2, 4, 6]
+    const boxes = [
+      [1, 1, 50, 50],
+      [19, 20, 30, 29],
+      [100, 20, 80, 100],
+      [500, 500, 80, 100]
+    ]
+
+    const trackIds = drawBox2DTracks(label2d, store, itemIndices, boxes)
+
+    // Terminate the track by button
+    dispatch(action.goToItem(2))
+    mouseMoveClick(label2d, 1, 30)
+    fireEvent(
+      getByText('Delete'),
+      new MouseEvent('click', {
+        bubbles: true,
+        cancelable: true
+      })
+    )
+    dispatch(action.goToItem(1))
+    mouseMoveClick(label2d, 1, 30)
+    let state = getState()
+    fireEvent(
+      getByText('Track-Link'),
+      new MouseEvent('click', {
+        bubbles: true,
+        cancelable: true
+      })
+    )
+
+    dispatch(action.goToItem(4))
+    Session.dispatch(selectLabel(
```

Comment:
Can we use mouse select here like the one line on this https://github.com/scalabel/scalabel/pull/126/files/edde618176e17d75002af65d54963403011d835e..ea102b95a423c06cdf44834dbd05a902995425c0#diff-2e1db3acd328930003a84bf0cd22cd9aR178?